### PR TITLE
Remove deprecated colours.

### PR DESCRIPTION
**Pull request by notlee**
- Remove branded palette tests which weren't running.
- Improve master brand palette tests.

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -31,16 +33,17 @@
...
+#### Palette Colours
 
-- `inherit`. Replace `oColorsByName('inherit');` with `inherit`.
-- `transparent`. Replace `oColorsByName('transparent');` with `transparent`.
+The following palette colours were removed for all brands:
+- `inherit`. Replace `oColorsGetPaletteColor('inherit');` with `inherit`.
+- `transparent`. Replace `oColorsGetPaletteColor('transparent');` with `transparent`.
 
-The following variables have been removed:
+Deprecated internal and whitelabel brand palette colours were removed. Please contact Origami if your internal brand product has a usecase for one of these colours:
```
Comment (notlee) [inline MIGRATION.md]:
```suggestion
Deprecated internal and whitelabel brand palette colours were removed. Please contact Origami if your internal product has a usecase for one of the removed colours colours:
```

File: MIGRATION.md [hunk 2]
```diff
@@ -31,16 +33,17 @@
...
+#### Palette Colours
 
-- `inherit`. Replace `oColorsByName('inherit');` with `inherit`.
-- `transparent`. Replace `oColorsByName('transparent');` with `transparent`.
+The following palette colours were removed for all brands:
+- `inherit`. Replace `oColorsGetPaletteColor('inherit');` with `inherit`.
+- `transparent`. Replace `oColorsGetPaletteColor('transparent');` with `transparent`.
 
-The following variables have been removed:
+Deprecated internal and whitelabel brand palette colours were removed. Please contact Origami if your internal product has a usecase for one of the removed colours colours:
```
Comment (notlee) [inline MIGRATION.md]:
```suggestion
Deprecated internal and whitelabel brand palette colours were removed. Please contact Origami if your product has a usecase for one of the removed colours colours:
```

**Comment by origamiserviceuser** [comment]
o-colors bundle size difference from 4.10.3 to e1808080392e8d48046d0b8348e3ded4e63a7ef5
css, master: 1.62kb decrease (-0.31kb/gzip)
css, internal: 2.2kb decrease (-0.44kb/gzip)
css, whitelabel: 3.9kb decrease (-0.73kb/gzip)

**Comment by origamiserviceuser** [comment]
o-colors bundle size difference from 4.10.3 to 59f8417b4bec23cb9f2e1422d0b4897649c6c014
css, master: 1.62kb decrease (-0.31kb/gzip)
css, internal: 2.2kb decrease (-0.44kb/gzip)
css, whitelabel: 3.9kb decrease (-0.73kb/gzip)

**Comment by origamiserviceuser** [comment]
o-colors bundle size difference from 4.10.3 to 65c61130c8cbeb6443ead5c2e73057c198a8c00b
css, master: 1.62kb decrease (-0.31kb/gzip)
css, internal: 2.2kb decrease (-0.44kb/gzip)
css, whitelabel: 3.9kb decrease (-0.73kb/gzip)

**Pull request merged**